CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# HOW TO CONTRIBUTE
+
+## Contribution Guide
+
+PairMesh is a community-driven open source project and we welcome any contributor.  Contributions to the PairMesh project are expected to adhere to our [Code of Conduct](https://github.com/pairmesh/pairmesh/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
PairMesh is a community-driven open source project and we look forward to you being part of the community of contributors. Contributors to ...